app/src/main/java/com/fstyle/structure_android/screen/userlist/UserListPresenter.java

```diff
@@ -0,0 +1,77 @@
+package com.fstyle.structure_android.screen.userlist;
+
+import android.support.annotation.NonNull;
+import android.util.Log;
```

Comment:
ERROR: Unused import: android.util.Log. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports